R/github-utils.R

```diff
@@ -1,24 +1,30 @@
-## suppress the warning about multiple remotes, which is triggered by
-## the common "origin" and "upstream" situation, e.g., fork and clone
-gh_tree_remote <- function(path) {
-  suppressWarnings(gh::gh_tree_remote(path))
+github_remotes <- function(path = proj_get()) {
+  remotes <- git_remotes(path)
+  if (length(remotes) == 0) return(NULL)
```

Comment:
What happened to official Jim style? 😛

Reply:
I thought these were OK:

```
if (length(remotes) == 0) return(NULL)

if (length(remotes) == 0) {
  return(NULL)
}
```

but this was not:

```
if (length(remotes) == 0)
  return(NULL)
```

@jimhester what is your ruling?

Reply:
@jennybc has it right, all on one line is ok since it is then not possible to add additional statements.

Reply:
Oh I really dislike that formatting 😞

Reply:
I never really do that myself, I always use braces and put it on a new line

Reply:
Braces and new lines, it is then!